src/main/java/org/chameleoncloud/representations/GlobusIdentity.java

```diff
@@ -0,0 +1,63 @@
+package org.chameleoncloud.representations;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class GlobusIdentity {
+
+    @JsonProperty("name")
```

Comment:
🤩

Reply:
This is much improved :)

Reply:
> It's really really nice how clean this ended up. Great job!
> 
> I am confused by what you mean here though:
> 
> > After digging into this a bit, I think we should actually hold off on adding this attribute mapper. I don't think it helps us in the authentication case as currently written.
> 
> Isn't the mapper the thing that's responsible for saving the Globus sub values as user attributes, so we can use them for authentication later?

Ah, this message was left over from the earlier commits, before the attributes were being checked by the authenticator.